function throwsException()
{
    throw new Exception('Exception message');
}
